### ISSUE TITLE
2.1.0 BGP peer regression

### ISSUE THREAD

**Issue by jojoflyhigh** [fortinet-ansible-dev/ansible-galaxy-fortios-collection#117]
Hello,
I have upgrade ansible module from 2.0.2 to 2.1.0.
I can no more create BGP peer and I have the following error.
![image](https://user-images.githubusercontent.com/36159893/123756019-cb3a9f00-d8bc-11eb-99b6-573656da161c.png)

**Comment by specialcircumstances** [comment]
I think I have a similar issue (potentially the same), so posting here...

In:
   fortios_router_bgp.py

def flatten_multilists_attributes(data):

I think this is failing and causing an error when the attributes are not defined. If I add the multi-line attributes as empty lists to the playbook definition of the neighbor then it works, but if they are missing there is a KeyError (for the non-required multi-line attribute).

```
          neighbor:
            - ip: "192.168.2.231"
              remote_as:  "65536"
              activate: "enable"
              activate6: "disable"
              attribute_unchanged: ""
              attribute_unchanged6: ""
```

when *missing* the empty lists for those attributes the stdout looks like this:

```
  File "/tmp/ansible_fortios_router_bgp_payload_q4t8_eag/ansible_fortios_router_bgp_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_router_bgp.py", line 10992, in <module>
  File "/tmp/ansible_fortios_router_bgp_payload_q4t8_eag/ansible_fortios_router_bgp_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_router_bgp.py", line 10971, in main
  File "/tmp/ansible_fortios_router_bgp_payload_q4t8_eag/ansible_fortios_router_bgp_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_router_bgp.py", line 2091, in fortios_router
  File "/tmp/ansible_fortios_router_bgp_payload_q4t8_eag/ansible_fortios_router_bgp_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_router_bgp.py", line 2074, in router_bgp
  File "/tmp/ansible_fortios_router_bgp_payload_q4t8_eag/ansible_fortios_router_bgp_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_router_bgp.py", line 2053, in flatten_multilists_attributes
  File "/tmp/ansible_fortios_router_bgp_payload_q4t8_eag/ansible_fortios_router_bgp_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_router_bgp.py", line 2043, in flatten_single_path
  File "/tmp/ansible_fortios_router_bgp_payload_q4t8_eag/ansible_fortios_router_bgp_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_router_bgp.py", line 2036, in flatten_single_path
KeyError: 'attribute_unchanged6'
```

Hope that's useful

**Comment by JieX19** [comment]
Hi @jojoflyhigh @specialcircumstances 

Thank you for finding the root cause of the issue. The bug is fixed in the latest version 2.1.1. 

Please install the new version and test it. 

Thanks!

**Comment by specialcircumstances** [comment]
Yup. Works me. Thanks for the swift response.
